doc/docs/NLopt_on_Windows.md

```diff
@@ -20,13 +20,28 @@ Unofficial Python binaries for Windows are available from Christoph Gohike:
 
 ### NLopt with MinGW
 
-If you want to compile NLopt on Windows with [MinGW](http://www.mingw.org/), be sure to install the MinGW version of `cmake` (e.g. with `pacman -S mingw-w64-x86_64-cmake`) and then build via `cmake -G"MSYS Makefiles" . && make` in order to ensure that `cmake` produces the correct type of makefile.
+If you want to compile NLopt on Windows:
+
+1. Install [MSYS2](https://www.msys2.org/).
+2. Run the following command:
+    pacman -S --needed base-devel mingw-w64-i686-toolchain mingw-w64-x86_64-toolchain \
+                    git subversion mercurial \
```

Comment:
Why do you need subversion and mercurial?

Reply:
I added them because I needed them! but I think git would be enough.